notifications/notificationmethod.py

```diff
@@ -37,7 +38,9 @@ class NotificationMethodPerformException(JobException):
 
 def _ssl_cert():
     if app.config["PREFERRED_URL_SCHEME"] == "https":
-        return [OVERRIDE_CONFIG_DIRECTORY + f for f in SSL_FILENAMES]
+        cert_files = [OVERRIDE_CONFIG_DIRECTORY + f for f in SSL_FILENAMES]
+        cert_exists = all([os.path.isfile(f) for f in cert_files])
+        return cert_files if cert_exists else None
```

Comment:
Would it be useful to log an info or even warning when `PREFERRED_URL_SCHEME` is https but the certificate files aren't there?

Reply:
I don't think so. Whether `PREFERRED_URL_SCHEME` is set and certificate files are missing or not are deployment specific, and aren't really unexpected behaviour. It would only create unnecessary noise every time a request is sent in my opinion.

Reply:
Alright, makes sense!